readme.md

```diff
@@ -317,6 +317,23 @@ const body = await got(url).json();
 const body = await got(url, {responseType: 'json', resolveBodyOnly: true});
 ```
 
+###### parseJson
+
+Type: `(text: string) => unknown`\
+Default: `(text) => JSON.parse(text)`
+
+Function used to parse JSON responses.
+
+Example:
```

Comment:
It's not immediately clear to the user what this example does different.

See https://github.com/sindresorhus/ky/pull/257/files#diff-0730bb7c2e8f9ea2438b52e419dd86c9R349-R367 for better docs.

Reply:
I can't figure out any meaningful example for `stringifyJson`, currently I added a random one that transforms all numbers in strings.

Reply:
A better example would be something that skips keys that begin with `_`.

Reply:
> A better example would be something that skips keys that begin with _.

Yes, in addition ot the current one with numbers.